### PR TITLE
Component version 1

### PR DESCRIPTION
**Pull request by Rupamdas832**
Modal is not working. Not fully responsive yet.

### CODE REVIEW

**Comment by gkanishk** [comment]
@Rupamdas832 can you provide a preview link for testing

File: src/Library/Card.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    <h5>New</h5>
+                </div>
+                <div className="cardImg">
+                    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRFxtc_LfP4RWXhrXJDDQyLZlEWVrlPLLhlF6LdU9UkiWlYGZfHEsSIzqU5fS7LGe6UOlHaCtpV&usqp=CAc"/>
+                </div>
+                <div className="cardBody">
+                    <p>boAt Airdopes 131 Bluetooth Headset</p>
+                <div className="cardPrice">
+                    <h4>₹5999</h4>
+                    <h5>50% off</h5>
```
Comment (priyagupta314-work) [inline src/Library/Card.js:105]: Hi Rupam, As per my understanding, h* tags are advisable only for headings. These can be replaced here, maybe a <div> tag. Let me know your thoughts.

**Review by priyagupta314-work — COMMENTED** [review]
Hi Rupam,
Few suggestions:
1. A little padding can be added to the footer.
2. Also, black text is not very clearly visible on the green background in footer, it can be made white or some other contrasting color.
3. The search box in the first navigation component is overflowing. It can be made smaller.
4. h* tags can be replaced if not being used for headings.
Rest LGTM. Great work in designing the components!!

**Comment by ritikarp** [comment]
Hey Rupam
Here are some pointers you can improve on,
 
- Make the side navigation bar fixed. 
- Maintain a consistent spacing and alignment throught your library.
- In inputs, avoid background color change to green, and style the Show button
- Remove extra spacing between social icons in the footer
- Can avoid round edges in navbar component at least for top, it won't often look good.
- Change the HTML title 

Rest LGTM. 
Highlights of the library for me is the hover effect on nav pills in side navigation. Amazing start @Rupamdas832

**Pull request merged**